vagrant/provisioning/roles/arkcase-app/templates/server.yml

```diff
@@ -116,3 +116,9 @@ calendar:
   :\"{{ calendar_purge_option }}\",\"daysClosed\":null}}"
 acm.hazelcast:
   log.folder: "{{ root_folder }}/log/arkcase"
+
+################### AWS Credentials (aws/credentials.properties) ####################
+## AccessKeyId and SecretAccessKey must be encrypted. ##
+aws.credentials:
+  awsAccessKeyId: {{ awsAccessKeyId_fact }}
+  awsSecretAccessKey: {{ awsSecretAccessKey_fact }}
```

Comment:
All you need is these two lines. Nothing more, no replacements needed, as Dave said.
Also, add the awsAccessKeyId_fact and awsSecretAccessKey_fact variables in arkcase-ce-facts.yml and  arkcase_dev_facts.yml with empty string values "", to serve as a documentation of possible facts you can define.